src/main/kotlin/commands/SlowCommand.kt

```diff
@@ -0,0 +1,46 @@
+package commands
+
+import Command
+import PermissionTypes
+import Permissions
+import StringHelper
+import StringHelper.MessageTypes.MISSING_PERMISSIONS
+import arg
+import doesLater
+import integer
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+object SlowCommand : Command("slow") {
+    init {
+        integer("wait") {
+            integer("time") {
+                doesLater { context ->
+                    if (!Permissions.hasPermission(message, PermissionTypes.COUNCIL_MEMBER)) {
+                        StringHelper.sendMessage(message.channel, MISSING_PERMISSIONS)
+                        return@doesLater
+                    }
+                    val wait: Int = context arg "wait"
+                    val time: Int = context arg "time"
+                    val originalWait = message.serverChannel!!.rateLimitPerUser
+                    message.serverChannel!!.edit {
+                        rateLimitPerUser = wait
```

Comment:
does this work with custom values such as 45 seconds?

Reply:
this should work with any value, although i didn't test past 30 seconds

Reply:
I'm asking because Discord's GUI limits you to specific intervals, are you sure it works with completely custom intervals?

Reply:
(tested with values like 2 5 30 20)